cityelf/src/main/java/com/cityelf/repository/IRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.cityelf.repository;
+
+import com.cityelf.model.Notification;
+import com.cityelf.model.User;
+
+/**
+ * Created by Sergii on 26.04.2017.
+ */
+public interface IRepository {
```

Comment:
This is not a java way of creating an interface. Do not use **I** before it.
Also just the word **Repository** means nothing. Change it to Notification repository.
Moreover you don't need to create an interface if you have just one implementation. You can easily add one if it's needed in the future by the means of IDE.